test/python/circuit/test_registerless_circuit.py

```diff
@@ -227,7 +227,7 @@ def test_circuit_barrier(self):
         circuit.barrier(range(0, 3))
 
         expected = QuantumCircuit(qreg01, qreg23)
-        expected.barrier([qreg01[0], qreg01[1], qreg23[0]])
+        expected.barrier(qreg01[0], qreg01[1], qreg23[0])
```

Comment:
We do supported, change undone in 33e81f9b.

Reply:
oh! but the reason for the change is because this is the `expected` circuit. So, it's the ground truth, as the standard. where only `expected.barrier(qreg01[0], qreg01[1], qreg23[0])` is supported. Let me know. I can undo `33e81f9` if you prefer.

Reply:
That makes sense, I think it's good to leave it as it is.

Reply:
the change 33e81f9 is undone in bfa7e639.... so back to `expected.barrier(qreg01[0], qreg01[1], qreg23[0])`